CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Unreleased
 
 ### Added
+- `SASRecModel` and `BERT4RecModel` - models based on transformer architecture ([#220](https://github.com/MobileTeleSystems/RecTools/pull/220))
+- Transfomers extended theory & practice tuotorial ([#220](https://github.com/MobileTeleSystems/RecTools/pull/220))
```

Comment:
```suggestion
- Transfomers extended theory & practice tutorial ([#220](https://github.com/MobileTeleSystems/RecTools/pull/220))
```

Usually we don't mention such things here since they are not a part of the package.  
But feel free to keep it if you want

Reply:
Extended theory & practice tutorials are always big stories. So I'd like to always mention them.
Training guide here is also important for the big picture.